packages/tasks/src/library-to-tasks.ts

```diff
@@ -31,6 +31,7 @@ export const LIBRARY_TASK_MAPPING: Partial<Record<ModelLibraryKey, PipelineType[
 	peft: ["text-generation"],
 	"pyannote-audio": ["automatic-speech-recognition"],
 	"sentence-transformers": ["feature-extraction", "sentence-similarity"],
+	"py-feat": ["image-feature-extraction"],
```

Comment:
As far as I know, this list is used for the inference snippet on the model pages. Since `py-feat` is not integrated with our inference service, we shouldn't list it here.

If you want `py-feat` models to be tagged as `image-feature-extraction`, you must tag them as such in the model card metadata (either using `pipeline_tag` or `tags`). See PR https://huggingface.co/py-feat/retinaface/discussions/1

Reply:
that makes sense. I'll remove it.